e2e-tests/constants/routes/your-business.js

```diff
@@ -0,0 +1,7 @@
+const INSURANCE_ROOT = '/insurance';
```

Comment:
Should this come from the insurance routes? would just need to add `export const` to the `INSURANCE_ROOT`

Reply:
causes a dependency cycle